packages/react-dev-utils/printFileSizes.js

```diff
@@ -0,0 +1,47 @@
+var fs = require('fs');
+var path = require('path');
+var chalk = require('chalk');
+var filesize = require('filesize');
+var stripAnsi = require('strip-ansi');
+var gzipSize = require('gzip-size').sync;
+
+// Print a detailed summary of build files.
+module.exports = paths => function printFileSizes(stats, previousSizeMap) {
```

Comment:
Instead of passing all `paths` let's just pass the `appBuild` path. That's the only that matters here. I don't want to deal with versioning whenever we change `paths` internal structure.